packages/common/src/types.ts

```diff
@@ -69,6 +69,8 @@ export const TranscriptionJob = z.object({
 	engine: z.nativeEnum(TranscriptionEngine),
 	// we can get rid of this when we switch to using a zip
 	translationOutputBucketUrls: z.optional(OutputBucketUrls),
+	// this is optional because giant currently doesn't know file duration
```

Comment:
do ffmpeg or the whisper models output file duration? Or would it be worth having the worker calculate duration with ffprobe? That would unify the way we're finding duration and make it so Giant isn't a special case.

Reply:
Good point. I was more just shoving this in in case useful for logging - the only place it was used was in setting a 'duration' field in the transcription failure message, which isn't really necessary. So I've removed this field in https://github.com/guardian/transcription-service/pull/136/commits/244a6d2d5dbf09b189c44363abd5643a972e1a6f